### PR TITLE
Improve demo styling

### PR DESCRIPTION
**Pull request by jtpio**
Some styling tweaks for the demo:

[jupyter-iframe-commands-demo-tweaks.webm](https://github.com/user-attachments/assets/c2670df2-7c0c-472d-b1e0-fffe432cb233)

### CODE REVIEW

**Comment by jtpio** [comment]
bot please update snapshots

**Comment by jtpio** [comment]
bot please update snapshots

File: demo/src/style.css [hunk 1]
```diff
@@ -97,9 +172,25 @@ dialog button[value='close']:hover {
...
+/* Responsive iframe */
+#jupyterlab {
+  width: 100%;
+  height: calc(100vh - 200px);
+  border: none;
+  border-radius: 8px;
+  margin-top: 1rem;
+  background-color: white;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+}
```
Comment (gjmooney) [inline demo/src/style.css:196]:
```suggestion
  align-self: center;
}
```

**Review by gjmooney — APPROVED** [review]
I just have some small suggestions. The scrollbar overlaps the iframe on Firefox, (and I thought it looked a little awkward with the iframe all the way against the edge) , so it could be nice to a little room on the sides. 

Here's what the scrollbar looks like in Firefox: 
scrollbar

**Comment by jtpio** [comment]
bot please update snapshots

**Comment by jtpio** [comment]
Thanks @gjmooney for taking a look!

Looks like it should be good to go now.

**Pull request merged**